apps/admin_app/lib/admin_app_web/views/product_view.ex

```diff
@@ -172,6 +172,20 @@ defmodule AdminAppWeb.ProductView do
     end
   end
 
+  def count_on_hand(conn) do
+    product_id = String.to_integer(conn.params["id"])
+
+    case StockItem.with_active_stock_location(product_id) do
+      # condition when product is created for the first time
+      [] ->
+        # and there are no stock items
+        0
+
```

Comment:
Remove this empty line as well. also the comments can be in one line imo.

Reply:
made the changes